web/satellite/src/components/account/billing/monthlySummary/MonthlyBillingSummary.vue

```diff
@@ -83,6 +83,10 @@ export default class MonthlyBillingSummary extends Vue {
      * chargesSummary returns summary of all projects.
      */
     public get chargesSummary(): number {
+        if (!this.usageCharges.length) {
```

Comment:
if (!this.usageCharges)

Reply:
Nikolai's variant was right, because he checked if array was empty (length === 0). If we use !this.usageCharges than we check if array exists at all. But it will always exist since we declared and initialized it earlier 'public usageCharges: ProjectCharge[] = this.$store.state.paymentsModule.charges;'. So your check will always return false even if array is empty

Reply:
Nikolai's variant is right, because he checks if declared earlier array is empty. But !this.usageCharges checks if array exists at all so it will always return false even if it's empty